src/lib.rs

```diff
@@ -220,20 +220,19 @@ impl MyDrone {
             });
             // there is no hops vec to reverse in sourcerouting header because
             // floodrequest ignores it, path trace is used instead
-            let hops: Vec<NodeId> = new_path_trace
-                .iter()
-                .map(|(id, _)| id)
-                .rev()
-                .copied()
-                .collect();
+
             let flood_response_packet = Packet {
                 pack_type: flood_response,
-                routing_header: SourceRoutingHeader { hop_index: 1, hops },
+                routing_header: SourceRoutingHeader { hop_index: 0, hops: vec![] },
```

Comment:
Flood Response uses the sourcerouting header to go back to sender, according to the protocol, it's just flood request that ignores it